packages/app/components/EventCard/EventCardText.js

```diff
@@ -48,29 +64,25 @@ function EventCardText(props) {
   return (
     <View>
       <Text style={styles.smallText}>
-        {props.timeBegin} - {props.timeEnd}
+        {formatDate(event.startDate)} - {formatDate(event.endDate)}
       </Text>
       <Text
         {...(isTitlePressed
           ? { style: styles.eventTitlePressed }
           : { style: styles.eventTitle })}
         onPress={onTitlePress}>
-        {props.title}
+        {event.title}
       </Text>
-      <Text style={styles.smallText}>📌 {props.location}</Text>
+      <Text style={styles.smallText}>📌 {event.location}</Text>
       <Text style={styles.description} numberOfLines={3}>
-        {props.description}
+        {event.description}
       </Text>
     </View>
   );
 }
 
 EventCardText.propTypes = {
-  description: PropTypes.string,
-  location: PropTypes.string,
-  timeBegin: PropTypes.string,
-  timeEnd: PropTypes.string,
-  title: PropTypes.string,
+  event: PropTypes.array,
```

Comment:
I am getting a Failed prop type: invalid prop 'event' of type 'object' supplied to 'EventCardText', expected 'array'

Make sure to handle any warnings you may see.

Reply:
👍 Changed prop type to any